src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -132,7 +132,7 @@ export function QueryPageContainerComponent(): JSX.Element {
             ...filter,
             selectedFilter: defaultFilter.selectedFilter,
         };
-        const newPath: string = generatePathStringService(newFilter, table);
+        const newPath: string = generatePathStringService(defaultFilter.selectedFilter, filter.mainFilter, table);

```

Comment:
Reject: mixed parameter source (prone to bugs). Please use: (newFilter.selectedFilter, newFilter.mainFilter)

Reply:
Done. Commit "Remove mixed parameter source & unused openProp"